integration-tests/steps.go

```diff
@@ -269,12 +271,199 @@ var happyPathSteps = []Step{
 		state: State{
 			chainID("consu"): ChainState{
 				ValPowers: &map[validatorID]uint{
-					validatorID("alice"): 500,
+					validatorID("alice"): 510,
+					validatorID("bob"):   500,
+					validatorID("carol"): 500,
+				},
+			},
+		},
+	},
+	{
+		action: SlashAction{
+			downOn: chainID("consu"),
+			// TODO: First validator cannot be brought down until this issue is resolved:
+			// https://github.com/cosmos/interchain-security/issues/263
+			toDown: validatorID("bob"),
+		},
+		state: State{
+			// validator should be slashed on consumer, powers not affected on either chain yet
+			chainID("provi"): ChainState{
+				ValPowers: &map[validatorID]uint{
+					validatorID("alice"): 510,
+					validatorID("bob"):   500,
+					validatorID("carol"): 500,
+				},
+			},
+			chainID("consu"): ChainState{
+				ValPowers: &map[validatorID]uint{
+					validatorID("alice"): 510,
+					validatorID("bob"):   500,
```

Comment:
@mpoke random question- i don't remember which way we ended up going on how the downtime slashing works, but just wanted to make sure that the power wasn't supposed to be reduced immediately as soon as the downtime happened on the consumer

Reply:
That's right. An infraction on the consumer causes a SlashRequest, which eventually causes a Slash on the provider, which eventually causes the power on the consumer to be reduced.